### DISCUSSION TITLE
Using created Inflow discharge as input

### DISCUSSION THREAD

**Discussion by Zhanwei-Liu** [iiasa/CWatM#167] [Q&A] [answered]
My calibration process involves two steps:

First, calibrating natural parameters using observed data from the historical period before dam construction.
Second, calibrating dam operation parameters. I am wondering if it is possible to save runoff results after calibrating natural parameters and then load them directly for the second step. If so, could you share insights on how to implement this?
I mean I only want to directly run routing module when calibrating reservoir operation parameters. Thanks a lot.

**Comment by mikhailsmilovic** [comment, marked as answer]
Guten Tag Zhanwei Liu

Yes, one can save discharge (instead of runoff) and have this as an inflow input. 

One first saves the discharge time series, for example, Gauges = 8.2 53.3 8.1 52

**Then to use this as an input:**
In section OPTIONS, include
inflow = True

In section INFLOW
InflowPoints = 8.25 49.75 7.75 50.25
InLocal = True     #only used if InflowPoints points to a map
QInTS = mm.tss

- If InflowPoints is a map, the InLocal flag is to identify if it is global (False) or local (True)
- observed or simulated input hydrographs as time series [cu m / s]
- Note: that identifiers in time series have to correspond to InflowPoints
- can be several timeseries in one file or different files e.g. main.tss mosel.tss

Let me know how it goes!

**Reply by Zhanwei-Liu** [reply]
Thank you for your response. I have tested this function, and it works well for me so far!
